addons/website_forum/static/src/js/website_forum.js

```diff
@@ -243,7 +260,8 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
         };
         if (session.is_website_user) {
             notifOptions.title = _t("Access Denied");
-            notifOptions.message = _t("Sorry you must be logged in to perform this action");
+            notifOptions.message =  Markup`
+            <p>Oh no! Please <a href='/web/login'>sign in</a> to vote</p>`;
```

Comment:
Why "oh no" ? Seems unnecessary bullshitfriendly addition ?

Reply:
I think we had some "oh no"s before too. Note that this code is cleaned in a later commit. This includes newline and translation.

Reply:
![image](https://github.com/odoo/odoo/assets/19491443/7467f770-e2c0-4c95-8d4b-cf737c869ef7)

Reply:
:vomiting_face: